src/content/en/fundamentals/performance/lazy-loading-guidance/images-and-video/index.md

```diff
@@ -0,0 +1,699 @@
+page load, and replaces them with lazily-loaded images as they're scrolled into
+the viewport.
+
+<figure>
+  <img srcset="images/lazy-loading-example-2x.jpg 2x,
+images/images/lazy-loading-example-1x.jpg 1x"
+src="images/images/lazy-loading-example-1x.jpg" alt="A screenshot of the website
+Medium in the browsing, demonstrating lazy loading in action. The blurry
+placeholder is on the left, and the loaded resource is on the right.">
+  <figcaption><b>Figure 2</b>. An example of image lazy loading in action. A
+placeholder image is loaded at page load (left), and when scrolled into the
+viewport, the final image loads at the time of need.</figcaption>
+</figure>
+
+If you're unfamiliar with lazy loading, you might be wondering just how useful
+the technique is, and what its benefits are. Read on to find out!
+
+## Why lazy load images or video instead of just _loading_ them?
+
+Because it's possible you're loading stuff the user may never see. This is
+problematic for a couple reasons:
+
+- It wastes data. On unmetered connections, this isn't the worst thing that could
+happen (although you could be using that precious bandwidth for downloading
+other resources that are indeed going to be seen by the user). On limited data
+plans, however, loading stuff the user never sees could effectively be a waste
+of their money.
+- It wastes processing time, battery, and other system resources. After a media
+resource is downloaded, the browser must decode it and render its content in the
+viewport.
+
+When we lazy load images and video, we reduce initial page load time, initial
+page weight, and system resource usage, all of which have positive impacts on
+performance. In this guide, we'll cover some techniques and offer guidance for
+lazy loading images and video, as well as [a short list of some commonly used
+libraries](/web/fundamentals/performance/lazy-loading-guidance/images-and-video/#lazy_loading_libraries).
+
+## Lazy loading images
+
+Image lazy loading mechanisms are simple in theory, but the details are actually
+a bit finicky. Plus there are a couple of distinct use cases that can both
+benefit from lazy loading. Let's first start with lazy loading inline images in
+HTML.
+
+### Inline images
+
+The most common lazy loading candidates are images as used in `<img>` elements.
+When we lazy load `<img>` elements, we use JavaScript to check if they're in the
+viewport. If they are, their `src` (and sometimes `srcset`) attributes are
+populated with URLs to the desired image content.
+
+#### Using intersection observer
+
+If you've written lazy loading code before, you may have accomplished your task
+by using event handlers such as `scroll` or `resize`. While this approach is the
+most compatible across browsers, modern browsers offer a more performant and
+efficient way to do the work of checking element visibility via [the
+intersection observer
+API](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API).
```

Comment:
It might be useful to contextually link to https://developers.google.com/web/updates/2016/04/intersectionobserver somewhere here.